html2text/utils.py

```diff
@@ -190,7 +190,7 @@ def skipwrap(para, wrap_links):
     # I'm not sure what this is for; I thought it was to detect lists,
     # but there's a <br>-inside-<span> case in one of the tests that
     # also depends upon it.
-    if stripped[0:1] == '-' or stripped[0:1] == '*':
+    if stripped[0:1] == '-' or (stripped[0:1] == '*' and not stripped[0:2] == '**'):
```

Comment:
This line is longer than 79 chars, which violates pep8 code quality style on line length.